spec/lib/data_update_scripts/update_tag_hotness_scores_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
```

Comment:
maybe we can skip this test as we're testing Rails in reality

Reply:
I was using it to ensure we still had a Tag AR model in the event that we remove it, I know HIGHLY unlikely. I'm game for removing it to save time. I added it to try and be "responsible" and test my new script 😉

Reply:
ahha our migrations and seed file would probably fail as well if that happens :D